tools/migrations/add-docker-index.js

```diff
@@ -8,47 +8,24 @@
  * Copyright (c) 2014, Joyent, Inc.
  */
 
-// Backfill image_uuid for KVM VMs
 var path = require('path');
-var fs = require('fs');
-var util = require('util');
 
 var bunyan = require('bunyan');
 var restify = require('restify');
-var moray = require('moray');
-var async = require('async');
-var levels = [bunyan.TRACE, bunyan.DEBUG, bunyan.INFO,
-              bunyan.WARN, bunyan.ERROR, bunyan.FATAL];
-var config;
-var log;
-
 
-/*
- * Loads and parse the configuration file at config.json
- */
-function loadConfig() {
-    var configPath = path.join(__dirname, '..', '..', 'config.json');
-
-    if (!fs.existsSync(configPath)) {
-        console.error('Config file not found: ' + configPath +
-          ' does not exist. Aborting.');
-        process.exit(1);
-    }
-
-    var theConfig = JSON.parse(fs.readFileSync(configPath, 'utf-8'));
-    return theConfig;
-}
+var configLoader = require('../../lib/config-loader');
+var moray = require('moray');
 
-var config = loadConfig();
+var configFilePath = path.join(__dirname, '..', '..', 'config.json');
+var config = configLoader.loadConfig(configFilePath);
 console.log(config);
```

Comment:
At least for consistency purposes, I think it would make more sense to bunyan log the config. My initial intention was to not change anything that wasn't related to the config loading itself. I can make the changes to bunyan log the config in that PR if we agree that's what we want to do.